Source/WebCore/platform/audio/UpSampler.h

```diff
@@ -42,8 +42,8 @@ class UpSampler final {
 public:
     explicit UpSampler(size_t inputBlockSize);
 
-    // The destination buffer |destP| is of size sourceFramesToProcess * 2.
-    void process(const float* sourceP, float* destP, size_t sourceFramesToProcess);
+    // The destination buffer |destination| is of size sourceFramesToProcess * 2.
```

Comment:
Why pass the size twice, then?

Reply:
Ok. I'll look into not passing the size and making sure that `source` has `size() == sourceFramesToProcess`